kani-driver/build.rs

```diff
@@ -9,5 +9,11 @@ fn main() {
     // https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates
     // https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-build-scripts
     // So "repeat" the info from build script (here) to our crate's build environment.
-    println!("cargo:rustc-env=TARGET={}", var("TARGET").unwrap());
+    let target_arch = var("TARGET").unwrap();
+    let workspace_root = var("CARGO_WORKSPACE_DIR").unwrap();
+    println!(
+        "cargo:rustc-env=KANI_EXTERN_OUT_DIR={}/target/{}/debug/deps",
```

Comment:
Same as before. Should we use https://doc.rust-lang.org/stable/std/path/struct.Path.html

Reply:
Also b835517aef9230827e606e92ac6d0e4929141d89.  However, I had to copy over the macro.